components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -76,17 +77,16 @@ abstract class EngineSession(
          * The engine received a request to load a request.
          *
          * @param url The string url that was requested.
+         * @param appIntent The Androio Intent that was requested.
          * @param triggeredByRedirect True if and only if the request was triggered by an HTTP redirect.
          * @param triggeredByWebContent True if and only if the request was triggered from within
          * web content (as opposed to via the browser chrome).
-         * @param shouldLoadUri Function to be called when request is consumed.
          */
         fun onLoadRequest(
-            url: String,
+            url: String?,
```

Comment:
Why did `url` on the EngineSession need to be nullable?

Reply:
You're right, it doesn't need to be null.  Same with the `appIntent`.  It was old code that should have been cleaned up.  I'll remove.